AddOns/BasicFinancials/app/src/codeunit/AssistedSetup.Codeunit.al

```diff
@@ -0,0 +1,46 @@
+codeunit 20605 "Assisted Setup BF"
+{
+    Access = Internal;
+
+    var
+        AssistedSetupTxt: Label 'Set up the Basic Financials extension';
+        AssistedSetupHelpTxt: Label 'https://go.microsoft.com/fwlink/?linkid=', Locked = true;
+        AssistedSetupDescriptionTxt: Label 'Basic Financials';
```

Comment:
this should explain what the extension will do, like "Set up the basic experience...". Brent would know to shape it better